src/app.py

```diff
@@ -0,0 +1,36 @@
+from quart import Quart, request
+from openai import AzureOpenAI
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+azure_endpoint = os.environ["AZURE_OPENAI_ENDPOINT"]
+azure_deployment = os.environ["AZURE_OPENAI_DEPLOYMENT"]
+
+app = Quart(__name__)
+
+@app.route("/api/chat", methods=['POST'])
+async def chat():
+    # Get the message from the request data
+    data = await request.get_json()
+    message = data["message"]
+
+    # gets the API Key from environment variable AZURE_OPENAI_API_KEY
+    client = AzureOpenAI(
```

Comment:
Since we're using Quart, we should take advantage of the Async client instead, I think?